internal/template/resources.go

```diff
@@ -104,6 +104,8 @@ var supportedResourceTypes = []struct {
 		},
 		templateAbleFields: [][]string{
 			{"spec", "resolverRef", "params", "[]", "value"},
+			{"spec", "contexts", "[]", "name"},
+			{"spec", "contexts", "[]", "description"},
```

Comment:
Should this be a name field?

Reply:
It's not a name field. It's a string with no other restrictions.
Also, the specific pattern that we're trying to support under this original request is `component_COMPONENT`. I.e. it should contain underscore after the literal `component` and before the name of the component, which does not comply with a name field.